### PR TITLE
Update community page

### PR DESCRIPTION
**Pull request by jahow**
Deprecated: IRC, developers mailing lists

Added: Github issue tracker, GIS Stackexchange

### CODE REVIEW

File: docsrc/community.rst [hunk 1]
```diff
@@ -3,51 +3,69 @@
...
+
+The `GeoNetwork GitHub issues tracker <https://github.com/geonetwork/core-geonetwork/issues>`_ is where all bug reports
+and feature requests should be done.
+
+There are templates for both uses that *needs to be followed* in order to facilitate
+reproduction (for bugs) or assessment (for features).
+
+Please note that anyone creating an issue is considered responsible for maintaining activity
+on it, for example by pinging community members or providing additional information, at least
+while no contributor has been specifically assigned to it.
```
Comment (Delawen) [inline docsrc/community.rst]: Issues with no activity in the last 6 months will be automatically closed. If it is an issue that is really important to you, help us in looking for funding to work on it.

Comment (Delawen) [reply docsrc/community.rst]: Adding something like that? So people don't complaint when we close issues.

Comment (Delawen) [reply docsrc/community.rst]: And then add something like https://github.com/bstriner/github-bot-close-inactive-issues somewhere.

**Comment by jahow** [comment]
Added that issues older that 6 months will be closed automatically.

@fgravin @fxprunayre are you ok with making public these new policies regarding GN community? Thanks

**Comment by Delawen** [comment]
Fixed the conflicts.

**Comment by jahow** [comment]
Thanks @Delawen :)

**Pull request merged**